package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@fluentui/fluentui-charting-contrib",
```

Comment:
move this package.json inside tools/UnitTestApp

Reply:
done